crates/turbopack-core/src/source_map/mod.rs

```diff
@@ -620,7 +600,20 @@ impl Serialize for CrateMapWrapper {
     fn serialize<S: Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
         use serde::ser::Error;
         let mut bytes = vec![];
-        self.0.to_writer(&mut bytes).map_err(Error::custom)?;
+        match &self.0 {
```

Comment:
Why do we want to change the serialization logic?

Reply:
I did it while debugging, because I wasn't sure the code used for source map generation